src/shared/index.tsx

```diff
@@ -196,10 +206,83 @@ ReactDOM.render(
       experiment={experiment3}
       config={mobileAppConfig}
     />
-    <h4>Experiment Schema JSON</h4>
-    <pre>
-      {JSON.stringify(experiment3, null, 2)}
-    </pre>
+    <ExpandoSchema experiment={experiment3} />
   </>,
   document.getElementById("experiment-3")
 );
+
+const experiment4 = {
+  "version": "1.0.0",
+  "metadata": {
+    "uuid": "e431af00-5ef9-44f8-a887-c76caa6ddde1",
+    "name": "Data Table Example",
+    "initials": "DT"
+  },
+  "schema": {
+    "sections": [
+      {
+        "title": "Collect",
+        "icon": "collect",
+        "formFields": ["experimentData"]
+      }
+    ],
+    "dataSchema": {
+      "type": "object",
+      "required": ["studySite", "label"],
+      "properties": {
+        "experimentData": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": [],
+            "properties": {
+              "location": {
+                "title": "Location",
+                "type": "string",
+                "readOnly": true
+              },
+              "trees": {
+                "title": "Tree Count",
+                "type": "number"
+              },
+              "leafColor": {
+                "title": "Leaf Color",
+                "type": "array",
+                "items": {
+                  "type": "string",
+                  "enum": ["Green", "Orange", "Red"]
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "formUiSchema": {
+      "experimentData": {
+        "ui:field": "dataTable",
+        "ui:dataTableOptions": {
+          "sensorFields": []
+        }
+      }
+    }
+  },
+  "data": {
+    "experimentData": [
```

Comment:
Are we missing an entry for Center here?

Reply:
This is for the shared component demo of the tables, not the real app.